src/EnergyPlus/DataLoopNode.hh

```diff
@@ -515,6 +517,7 @@ namespace DataLoopNode {
         "Dehumidifier:Desiccant:System",
```

Comment:
These `constexpr std::array<std::string_view>` arrays should be moved to the .cc` files. If they are needed outside of this module you can leave an `extern const` declaration here. [Sorry, this is a new thing.]